test/integration/integration_test.go

```diff
@@ -1375,22 +1375,22 @@ func testBackupSpecifiedPartition(r *require.Assertions, ch *TestClickHouse) {
 
 	testBackupName := fmt.Sprintf("test_backup_%d", rand.Int())
 	// Create table
-	ch.queryWithNoError(r, "CREATE TABLE default.t1 (dt DateTime, v UInt64) ENGINE=MergeTree() PARTITION BY toYYYYMMDD(dt) ORDER BY dt")
-	ch.queryWithNoError(r, "INSERT INTO default.t1 SELECT '2022-01-01 00:00:00', number FROM numbers(10)")
-	ch.queryWithNoError(r, "INSERT INTO default.t1 SELECT '2022-01-02 00:00:00', number FROM numbers(10)")
+	ch.queryWithNoError(r, "CREATE TABLE default.t1_ (dt DateTime, v UInt64) ENGINE=MergeTree() PARTITION BY toYYYYMMDD(dt) ORDER BY dt")
```

Comment:
You continue to doing stupid things

instead of try to do my suggestions
please STOP